.changeset/witty-taxis-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@redocly/cli': patch
+---
+
+Set default severity for the config validation.
```

Comment:
Please include why this is useful, and what behaviour change the user might experience.

Reply:
Updated